pokedex/src/App.css

```diff
@@ -1,46 +1,113 @@
+html,
+body {
+  height: 100%;
+  width: 100%;
+}
+.flex-container > * {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+@media all and (min-width: 1232px) {
+  .top-flex-1 {
+    order: 1;
+  }
+  .title {
+    order: 2;
+  }
+  .top-flex-wide {
+    order: 3;
+  }
+  .right {
+    order: 4;
+  }
+}
+@media all and (max-width: 1232px) {
+  .title {
+    flex-grow: 1;
+  }
+  .flex-container {
+    justify-content: space-around;
+  }
+}
```

Comment:
This isn't something I want you to change, just a good thing to call out. All the widths here are using somewhat arbitrary numbers. There are somewhat well defined breakpoints that we typically use when styling for different mobile widths. 

Here are a couple examples:

[](https://www.w3schools.com/howto/howto_css_media_query_breakpoints.asp)

[](https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/)

Often times using a framework is the best way to deal with these different sizes.